src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+
+  if (days < 3) {
+    return days * dayCost;
+  };
+
+  if (days < 7) {
+    return days * dayCost - 20;
```

Comment:
It would be better to create a const variable for  20 as well

Reply:
done